### ISSUE TITLE
Remove the ability to save when in combat

### ISSUE THREAD

**Issue by GoogleCodeExporter** [alistairreilly/andors-trail#241]
```
In order to reduce the possibility of cheese tactics, we should remove the 
option of doing the save (menu->save) when engaged in combat. Autosave should 
of course still work, since it's a vital part of an Android app, but the manual 
save should be disabled IMO.

Probably with a message stating that "You cannot save while in combat".
```

Original issue reported on code.google.com by `oskar.wi...@gmail.com` on 5 Sep 2011 at 8:27

**Comment by GoogleCodeExporter** [comment]
```
I agree with being able to save mid battle as it means you can just abuse it by 
saving and reloading after each hit. However in no rpg should an autosave be 
present, especially in rpgs. Example you buy some expensive new equipment and 
then the game autosaves, and them you realise you don't like it. Autosaves just 
take you out of control. Manual save is better and should not be turned off
```

Original comment by `nezkeys1...@gmail.com` on 6 Sep 2011 at 3:45

**Comment by GoogleCodeExporter** [comment]
```
I agree with being able to save mid battle as it means you can just abuse it by 
saving and reloading after each hit. However in no rpg should an autosave be 
present, especially in rpgs. Example you buy some expensive new equipment and 
then the game autosaves, and them you realise you don't like it. Autosaves just 
take you out of control. Manual save is better and should not be turned off
```

Original comment by `nezkeys1...@gmail.com` on 6 Sep 2011 at 3:47

**Comment by GoogleCodeExporter** [comment]
```
Implemented in r165.
```

Original comment by `oskar.wi...@gmail.com` on 5 Oct 2011 at 10:10
- Changed state: **Ready**
- Added labels: **Milestone-0.6.10**

**Comment by GoogleCodeExporter** [comment]
```
Released in v0.6.10 . Closing.
```

Original comment by `oskar.wi...@gmail.com` on 21 Oct 2011 at 8:56
- Changed state: **Fixed**
